modules/updating-an-accelerator-profile.adoc

```diff
@@ -7,25 +7,49 @@
 You can update the existing accelerator profiles in your deployment. You might want to change important identifying information, such as the display name, the identifier, or the description. 
 
 .Prerequisites
-* You have logged in to {openshift-platform}.
-* You have the `cluster-admin` role in {openshift-platform}.
-* The `AcceleratorProfile` custom resource definition (CRD) exists in your deployment and has an accelerator profile.
+* You have logged in to {productname-long}.
+ifndef::self-managed[]
+* You are part of the `dedicated-admins` user group in {openshift-platform}.
+endif::[]
+ifdef::self-managed[]
+* You are assigned the `cluster-admin` role in {openshift-platform}.
+endif::[]
+* The accelerator profile exists in your deployment.
 
 .Procedure
-. In the {openshift-platform} web console, in the *Administrator* perspective, click *Administration* -> *CustomResourceDefinitions*.
-. In the search bar, enter `acceleratorprofile` to search by name.
+. From the {productname-short} dashboard, click *Settings* -> *Notebook images*.
 +
-The *CustomResourceDefinitions* page reloads to display the search results.
+The *Notebook image settings* page appears. Previously imported notebook images are displayed. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enable* column.
+. Click the action menu (&#8942;) and select *Edit* from the list.
 +
-. Click the `AcceleratorProfile` custom resource definition (CRD).
+The *Edit accelerator profile* dialog opens.
```

Comment:
Having 'if applicable' on all of these steps feels a bit unnecessary, could remove from each.